include/bluetooth/bluetooth.h

```diff
@@ -297,21 +349,46 @@ enum {
 
 	/** Use whitelist to filter devices that can connect. */
 	BT_LE_ADV_OPT_FILTER_CONN = BIT(7),
+
+	/* Generate scan reports when advertiser was scanned by active scanner.
+	 */
+	BT_LE_ADV_OPT_SCAN_REPORTS = BIT(8),
+
+	/* Support scan response. */
+	BT_LE_ADV_OPT_SCANNABLE = BIT(9),
+
+	/** Advertise with extended data length. */
+	BT_LE_ADV_OPT_EXT_DATA = BIT(10),
+
+	/* Primary Advertising on coded PHY */
+	BT_LE_ADV_OPT_EXT_DATA_NO_2M = BIT(11),
+
+	/* Secondary Advertising on coded PHY. */
+	BT_LE_ADV_OPT_LONG_RANGE = BIT(12),
```

Comment:
@rugeGerritsen @jhedberg @joerchan 

>BT_LE_ADV_OPT_LONG_RANGE

I completely disagree with using `LONG_RANGE` in the API for the following reasons:
- It is not in the spec at all (except when listing the features added in a particular spec version
- Almost all API users end up needing to check with the spec, and they won't find the mapping between this and coded phy
- Coded phy has potential uses beyond long range (better noise resilience)
- All  sniffer and HCI logs will use CODED instead